src/vs/workbench/contrib/notebook/browser/view/renderers/webviewPreloads.ts

```diff
@@ -869,7 +869,7 @@ async function webviewPreloads(style: PreloadStyles, options: PreloadOptions, re
 					const state = vscode.getState();
 					return typeof state === 'object' && state ? state[id] as T : undefined;
 				},
-				getRenderer: (id: string) => renderers.getRenderer(id)?.api,
+				getRenderer: async (id: string) => renderers.getRenderer(id)?.api,
```

Comment:
I would add a comment here why we're returning a promise (presumably in case internals are changed in the future) for a synchronous value (which is safe because currently we activate the module prior to load its dependents)